database/src/main/resources/db/migration/V118__klagevedtakshistorikk.sql

```diff
@@ -0,0 +1,2 @@
+alter table klage
+    add column if not exists klagevedtakshistorikk jsonb not null default '[]'::jsonb
```

Comment:
Nå kommer jeg kanskje litt seint til festen, men denne dataen har vi vel allerede i klagevedtak-tabellen og kunne vel bare joinet inn den? :grimacing:

Reply:
Hehe jepp, kunde joinet den men tänkte att ikke alla vedtak kommer nødvändigtvis fra KA. Så spørsmål er hvordan man har lust å strukturere opp datamodellen här.

1. Man kan lagre vedtaken i separata databaser og joina de etterpå og sortere etter opprettet-tidspunkt
2. Man kan putte in de fattede vedtaken in i en jsonb på `klage`:n etter de er prosessert/håndtert.